app/src/main/java/com/example/mycalculator/mvp/CalculatorPresenterImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorPresenterImpl implements CalculatorContract.calculatorPresenter {
+
+    private CalculatorModelImpl model;
+    private CalculatorViewImpl view;
+
+
+    public CalculatorPresenterImpl(CalculatorModelImpl model, CalculatorViewImpl view) {
```

Comment:
same here, the type should be the interface

Reply:
Done. Thx!